framework/src/org/checkerframework/common/value/ValueCheckerUtils.java

```diff
@@ -97,7 +98,8 @@
         } else if (AnnotationUtils.areSameByClass(anno, BottomVal.class)) {
             values = convertBottomVal(anno, castType);
         } else if (AnnotationUtils.areSameByClass(anno, UnknownVal.class)
-                || AnnotationUtils.areSameByClass(anno, ArrayLen.class)) {
+                || AnnotationUtils.areSameByClass(anno, ArrayLen.class)
+                || AnnotationUtils.areSameByClass(anno, IntRange.class)) {
```

Comment:
This looks wrong to me.  If the `@IntRange` annotation spans 10 or fewer elements, then this should return a list of those elements.  Otherwise, various functionality that works for `@IntVal` annotations will not work for `@IntRange` annotations, such as compile-time execution of `@StaticallyExecutable` methods.